docs/src/openapi-v1.yaml

```diff
@@ -20,6 +20,98 @@ tags:
+        specific to the assets-pallet for parachains. If no `assets` query parameter
+        is provided, all asset-balances for the given account will be returned. 
+      operationId: getAssetBalances
+      parameters:
+      - name: accountId
+        in: path
+        description: SS58 address of the account.
+        required: true
+        schema:
+          type: string
+          pattern: '^[1-9A-HJ-NP-Za-km-z]{8,64}$'
+      - name: at
+        in: query
+        description: Block at which to query asset-balance info for the 
+          specified account.
+        required: false
+        schema:
+          type: string
+          description: Block height (as a non-negative integer) or hash
+            (as a hex string).
+          format: unsignedInteger or $hex
+      - name: assets
+        in: query
+        description: An array of AssetId's to be queried. If not inputed by default
+          all asset-balances associated with the `accountId` will be returned. The array
+          format follows Express 4.x API recommendations. ex:`?assets[]=1&assets[]=2&assets[]=3`.
+        required: false
+        schema:
+          type: string
+          description: An array of assetId numbers represented as strings
+          format: Array of unsignedInteger's
+      responses:
+        "200": 
+          description: successfull operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AccountAssetsBalances'
+  /accounts/{accountId}/asset-approvals:
+    get:
+      tags:
+      - accounts
+      summary: Get an asset-approval for an account.
+      description: Returns information about an account's asset-approval transaction.
+        It is required to pass in a delegate and an assetId as query parameters.
+      operationId: getAssetApprovals
+      parameters:
+      - name: accountId
+        in: path
+        description: SS58 address of the account.
+        required: true
+        schema:
+          type: string
+          pattern: '^[1-9A-HJ-NP-Za-km-z]{8,64}$'
+      - name: at
+        in: query
+        description: Block at which to query asset-approval info for the 
+          specified account.
+        required: false
+        schema:
+          type: string
+          description: Block height (as a non-negative integer) or hash
+            (as a hex string).
+          format: unsignedInteger or $hex
+      - name: assetId
+        in: query
+        description: The `assetId` associated with the asset-approval.
+        required: true
+        schema:
+          type: string
+          description: An assetId represented as an unsignedInteger.
+          format: unsignedInteger
+      - name: delegate
+        in: query
+        description: The delegate's `accountId` associated with an asset-approval.
+        required: true
+        schema:
+          type: string
+          pattern: '^[1-9A-HJ-NP-Za-km-z]{8,64}$'
```

Comment:
@dvdplm does this look like an ok regex?

Reply:
Awesome in this case, i switched from `pattern: <some-regex>` to `format: SS58`. Per my conversation with @emostov it seems as if regex is not the best way to verify SS58's pattern via the docs but instead to just specifiy it's format.